# request message
uint8 robot_position
uint8 motion_type
uint8 rotation_type
uint8 calibration_duration
HSVThresholds hsv_thresholds
uint16 camera_width
uint16 camera_height
---
# response message
CalibrationPoint[] calibration_points_left_arm
CalibrationPoint[] calibration_points_right_arm
